frontend/src/selectors/reportSelectors.js

```diff
@@ -2,3 +2,6 @@ import * as reportReducer from "@/reducers/reportReducer";
 
 // eslint-disable-next-line import/prefer-default-export
 export const { getMineReports } = reportReducer;
+export const { getMineReportComments } = reportReducer;
+export const { getMineReportCommentLoading } = reportReducer;
+export const { getMineReportCommentSubmitting } = reportReducer;
```

Comment:
`eslint-disable` comment can be removed here as well 🙂

Reply:
Removed